<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterPage</class>
 <widget class="QDialog" name="RegisterPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Register </string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/splash/xilften.png</normaloff>:/splash/xilften.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="6">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>119</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(185, 0, 0);</string>
     </property>
     <property name="text">
      <string>Registration</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>119</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>119</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="BackButton">
     <property name="font">
      <font/>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgb(204, 0, 0);
   	border-radius:5px;
}
QPushButton:hover{
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(255, 255, 255);
}
</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
     <property name="icon">
      <iconset resource="image.qrc">
       <normaloff>:/icons/icons/back.png</normaloff>:/icons/icons/back.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="RegUsername">
       <property name="font">
        <font/>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(255, 255, 255);
border-radius:5px;</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="RegPass">
       <property name="font">
        <font/>
       </property>
       <property name="styleSheet">
        <string notr="true">

background-color: rgb(255, 255, 255);
border-radius:5px;</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="RegFullName">
       <property name="font">
        <font/>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(255, 255, 255);
border-radius:5px;</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Full Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="RegEmail">
       <property name="font">
        <font/>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(255, 255, 255);
border-radius:5px;</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Email</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="RegAge">
       <property name="font">
        <font/>
       </property>
       <property name="styleSheet">
        <string notr="true">

background-color: rgb(255, 255, 255);
border-radius:5px;</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Age</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="LogBut">
       <property name="font">
        <font/>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgb(204, 0, 0);
   	border-radius:5px;
}
QPushButton:hover{
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(255, 255, 255);
}
</string>
       </property>
       <property name="text">
        <string>Register</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/icons/icons/reg.png</normaloff>:/icons/icons/reg.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="RegisterOk">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(185, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>119</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="4">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
